src/Compression/Compression.cs

```diff
@@ -547,10 +555,13 @@ protected override void Dispose (bool disposing)
 			}
 		}
 
+#if !DOTNET_TODO
+		// DOTNET_TODO: See comment about TaskToApm elsewhere in this file.
 		public override IAsyncResult BeginWrite (byte[] array, int offset, int count, AsyncCallback asyncCallback, object asyncState) =>
 			TaskToApm.Begin(WriteAsync (array, offset, count, CancellationToken.None), asyncCallback, asyncState);
 
 		public override void EndWrite (IAsyncResult asyncResult) => TaskToApm.End (asyncResult);
+#endif // !DOTNET_TODO
```

Comment:
Same, I think it is safe to remove since we do not change the API. is less efficient but users should have moved (if we do have any of them in this API).